src/main/java/be/polyscripts/contactmanagerapp/api/FrontApi.java

```diff
@@ -0,0 +1,49 @@
+package be.polyscripts.contactmanagerapp.api;
+
+import be.polyscripts.contactmanagerapp.dto.AuthenticationRequest;
+import be.polyscripts.contactmanagerapp.dto.AuthenticationResponse;
+import be.polyscripts.contactmanagerapp.model.User;
+import be.polyscripts.contactmanagerapp.service.UserService;
+import be.polyscripts.contactmanagerapp.utils.JwtUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+public class FrontApi {
```

Comment:
add:
@RequestMapping(value = "/api/v1/contacts") or some thing like that to distingue your API versions and paths

Reply:
done !